docs/sources/migration-guide/migrating-from-thanos-or-prometheus.md

```diff
@@ -47,53 +47,35 @@ aws s3 cp <DIRECTORY> s3://<TENANT>/<DIRECTORY>
 gsutil -m cp -r <DIRECTORY> gs://<TENANT>/<DIRECTORY>
 ```
 
-## Migrating the block `meta.json` metadata using `metaconvert`
+## Block metadata
 
-Every block has a `meta.json` metadata file used by Grafana Mimir, Prometheus, and Thanos to identify the block contents.
+Each block has a `meta.json` metadata file that is used by Grafana Mimir, Prometheus, and Thanos to identify the block contents.
 Each project has its own metadata conventions.
-The `metaconvert` tool migrates the `meta.json` metadata from one project to another.
 
-### Downloading `metaconvert`
+In the Grafana Mimir 2.1 (or earlier) release, the ingesters added an external label to the `meta.json` file to identify the tenant that owns the block.
+If you enable the sharding feature of Mimir's split-and-merge compactor, compactor adds a special label to identify the compactor-shard.
 
-- If you are using Docker, run the following command:
+In the Grafana Mimir 2.2 (or later) release, blocks no longer have a label that identifies the tenant. The compactor-shard label is still used, if sharding of blocks during compaction is enabled.
 
-```bash
-docker pull grafana/metaconvert:latest
-```
-
-- If you are using a release binary, download the appropriate release asset for your operating system and architecture and make it executable. For Linux with the AMD64 architecture, run the following command:
-
-```bash
-curl -LO https://github.com/grafana/mimir/releases/latest/download/metaconvert-linux-amd64
-chmod +x metaconvert-linux-amd64
-```
-
-- If you are using Go, run the following command:
-
-```bash
-go install github.com/grafana/mimir/cmd/metaconvert@latest
-```
-
-### Running `metaconvert`
-
-> **Warning:** The `metaconvert` tool modifies objects in place.
-> Ensure you enable bucket versioning or have backups before running the tool.
-
-To run `metaconvert`, provide it with the bucket configuration. Use `metaconvert -h` to list available parameters.
+> **Note**: Blocks from Prometheus do not have any external labels stored in them; only blocks from Thanos use labels.
 
-1. Complete one of the following steps to run `metaconvert` in a dry-run mode, which lists blocks for migration.
+## Considerations on Thanos specific features
 
-   - If you are using Docker, run the following command:
+> **Note**: Thanos requires that Prometheus is configured with external labels.
```

Comment:
Looks a bit weird to me starting the section with a "note" (so far I've seen notes being addendum at the end of a section). I would consider making these sentences normal ones, not a "note".